edgeless_orc/src/proxy_redis.rs

```diff
@@ -56,11 +56,11 @@ impl ProxyRedis {
         );
 
         // create the connection with the Redis server
-        let mut connection = redis::Client::open(redis_url)?.get_connection()?;
+        let mut connection: redis::Connection = redis::Client::open(redis_url)?.get_connection()?;
```

Comment:
Is there a specific reason why you forced the return type here?

I tried reverting to the original code and I get no warning with the latest compiler (`rustc 1.82.0 (f6e511eec 2024-10-15)`)

Reply:
Not really, it was the first thing I tried when trying to fix the warning.
You are right, it is not needed, I'll revert the change